explorer/src/Pos/Explorer/ExplorerMode.hs

```diff
@@ -107,22 +108,26 @@ type ExplorerTestMode = ReaderT ExplorerTestContext Emulation
 type ExplorerExtraTestMode = ExtraContextT ExplorerTestMode
 
 data ExplorerTestContext = ExplorerTestContext
-    { etcGState       :: !GS.GStateContext
-    , etcSystemStart  :: !Timestamp
-    , etcSSlottingVar :: !SimpleSlottingStateVar
-    , etcSlotId       :: !(Maybe SlotId)
+    { etcGState              :: !GS.GStateContext
+    , etcSystemStart         :: !Timestamp
+    , etcSSlottingVar        :: !SimpleSlottingStateVar
+    , etcSlotId              :: !(Maybe SlotId)
```

Comment:
This is why I am against vertical alignment. This change is semanically null.

Reply:
strong agree